### PR TITLE
Auto clear lootruns on delete

### PR DESCRIPTION
**Pull request by PhoenixOrigin**
Pretty simple, just a small addition that to my understanding should clear the lootrun shown if you use delete.

### CODE REVIEW

**Comment by DevChromium** [comment]
Shouldn't there also be a check if the lootrun you want to delete is the active one?
If I'm doing a lootrun and I decide to delete a random one in my list, this piece of code would unload my current lootrun.

File: src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java [hunk 1]
```diff
@@ -129,7 +129,12 @@ public void execute(MinecraftServer server, ICommandSender sender, String[] args
                 if (result) {
                     message = GREEN + "Deleted run " + name + " successfully!";
                 } else {
-                    message = RED + "The provided lootrun doesn't exists!";
+                    sender.sendMessage(new TextComponentString(RED + "The provided lootrun doesn't exists!"));
```
Comment (kristofbolyai) [inline src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java]: No need to change this line, we print message later on without the return

File: src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java [hunk 2]
```diff
@@ -129,7 +129,12 @@ public void execute(MinecraftServer server, ICommandSender sender, String[] args
...
+                    return;
```
Comment (kristofbolyai) [inline src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java]: This line breaks code. Remove it

File: src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java [hunk 3]
```diff
@@ -129,7 +129,12 @@ public void execute(MinecraftServer server, ICommandSender sender, String[] args
...
+                }
+                
+                if (LootRunManager.getActivePath() != null) {
```
Comment (kristofbolyai) [inline src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java]: Check if activepath is the deleted one, only clear if so.

Basically just put this into the if above and it should work.

**Review by kristofbolyai — CHANGES_REQUESTED** [review]

File: src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java (hunk 2 above)
Comment (PhoenixOrigin) [reply src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java]: It was intended to stop the further code from running in the case it doesn't exist and in turn clearing the active lootrun. So what does it break apart from what I made it do?

File: src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java (hunk 3 above)
Comment (PhoenixOrigin) [reply src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java]: Ok 👍

File: src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java (hunk 1 above)
Comment (PhoenixOrigin) [reply src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java]: As shown in the message below

File: src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java (hunk 2 above)
Comment (kristofbolyai) [reply src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java]: Yes I know what return does but if you move the active path if check (the one below) to the correct place this return is not needed and the line above does not need to be changed

Comment (PhoenixOrigin) [reply src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java]: Ok

File: src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java [hunk 4]
```diff
@@ -129,7 +129,12 @@ public void execute(MinecraftServer server, ICommandSender sender, String[] args
                 if (result) {
                     message = GREEN + "Deleted run " + name + " successfully!";
                 } else {
-                    message = RED + "The provided lootrun doesn't exists!";
+                    sender.sendMessage(new TextComponentString(RED + "The provided lootrun doesn't exists!"));
+                    return;
+                }
+                
+                if (LootRunManager.getActivePath() != null && LootRunManager.getActivePathName().equals(name) {
```
Comment (kristofbolyai) [inline src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java]: If missing a )?

**Review by kristofbolyai — CHANGES_REQUESTED** [review]

File: src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java [hunk 5]
```diff
@@ -128,6 +128,9 @@ public void execute(MinecraftServer server, ICommandSender sender, String[] args
                 String message;
                 if (result) {
                     message = GREEN + "Deleted run " + name + " successfully!";
+                    if (LootRunManager.getActivePath() != null && LootRunManager.getActivePathName().equals(name)) {
+                        LootRunManager.clear();
+                    }
                 } else {
                     message = RED + "The provided lootrun doesn't exists!";
```
Comment (ryanzhoudev) [inline src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java]:
```suggestion
                    message = RED + "The provided lootrun doesn't exist!";
```

File: src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java (hunk 4 above)
Comment (PhoenixOrigin) [reply src/main/java/com/wynntils/modules/map/commands/CommandLootRun.java]: done

**Review by ryanzhoudev — APPROVED** [review]

**Review by Incompleteusern — APPROVED** [review]

**Review by kristofbolyai — APPROVED** [review]

**Pull request merged**